build-tests-samples/heft-storybook-react-tutorial-storykit/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "heft-storybook-react-tutorial-storykit",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Storybook build dependencies for heft-storybook-react-tutorial",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
```

Comment:
Yes, they are handcoded shims that simply reexport `export * from '@storybook/react'`. It could be TypeScript source code, but setting up Heft seemed like overkill for such a trivial output.